docs/src/examples/components/SplitButton/Types/SplitButtonExampleSmall.shorthand.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import { SplitButton } from '@stardust-ui/react'
+
+const SplitButtonExampleSmallShorthand = () => (
+  <>
+    <SplitButton
+      menu={[
+        { key: 'group', content: 'New group message' },
+        { key: 'channel', content: 'New channel message' },
+      ]}
+      button={{
+        content: 'small',
+        'aria-roledescription': 'splitbutton',
+        'aria-describedby': 'instruction-message',
+      }}
+      small
+      toggleButton={{ 'aria-label': 'options' }}
```

Comment:
label should be a bit more descriptive, like `more options` or `expand options menu` ...